### DISCUSSION TITLE
SelectedItems in DataGrid

### DISCUSSION THREAD

**Discussion by IngvarX** [AvaloniaUI/Avalonia#5694] [Q&A] [answered]
Hello,

Do you have any plans to make `SelectedItems` property of `DataGrid` bindable? For now I have do some kind of manual binding which is annoying.

**Comment by maxkatz6** [comment, marked as answer]
I don't think it's planned to be implemented.
And as far as I know, WPF/Silverligh/UWP data grids is not bindable in that way either, unfortunately, so it will require deeper knowledge of that control.
